java/compaction-status-store/src/main/java/sleeper/compaction/dynamodb/job/DynamoDBCompactionJobStatusFormat.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2022 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package sleeper.compaction.dynamodb.job;
+
+public class DynamoDBCompactionJobStatusFormat {
+
+    private DynamoDBCompactionJobStatusFormat() {
+    }
+
+    public static final String JOB_ID = "JobId";
+    public static final String LAST_UPDATE_TIME = "LastUpdateTime";
```

Comment:
Would this be better named UPDATE_TIME rather than LAST_UPDATE_TIME? When we put entries into the status store, we'll end up with multiple entries for each job, each of which will have the time of the update and a value. But it's not really the last update time? The status store lets us see the lifecycle of the job by creating a time-ordered list of the status updates.

Reply:
Done